linux/opt/sage/bin/apache_conf_rewrite.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
-. /opt/sage/bin/instance_env_vars.sh  && sed -i "s/EC2_INSTANCE_ID/$EC2_INSTANCE_ID/g" /etc/apache2/sites-available/proxy.conf
+. /opt/sage/bin/instance_env_vars.sh  && sed -i "s/^.*<LocationMatch.*\/.*\/>.*$/<LocationMatch \/$EC2_INSTANCE_ID\/>/g" /etc/apache2/sites-available/proxy.conf
```

Comment:
i would prefer to decouple one bash script from the other by running `instance_env_vars.sh` from the CFN template, like 
```
/opt/sage/bin/instance_env_vars.sh && opt/sage/bin/apache_conf_rewrite.sh
```

Reply:
My script doesn't actually 'run' the first bash script... `instance_env_vars.sh` is the output of a script that runs earlier... and `instance_env_vars.sh` just exports variables like `EC2_INSTANCE_ID=i-12345xyztvfpppppp`. I need to `source` (ie, `.`) `instance_env_vars.sh` in order to get the value of `EC2_INSTANCE_ID`

Reply:
yes, i understand that.  i still prefer to de-couple the two.